debian/rules

```diff
@@ -26,4 +26,8 @@ override_dh_auto_test:
 override_dh_auto_install:
 	mkdir debian/tmp
 
+# Don't start - monit does that for us.
+override_dh_installinit:
+	dh_installinit --no-start -u"defaults 60 40"
```

Comment:
Might be tempted to specify the package, otherwise this might apply to new packages accidentally.